frontend/src/app/document-view/document-view.component.html

```diff
@@ -18,7 +18,10 @@
                     <ng-container *ngFor="let field of propertyFields">
                         <tr *ngIf="document.fieldValue(field)">
                             <th [iaBalloon]="field.description" iaBalloonPosition="up" iaBalloonLength="medium">{{field.displayName}}</th>
-                            <td *ngIf="!isUrlField(field)" data-test-field-value [innerHtml]="highlightedInnerHtml(field)"></td>
+                            <td *ngIf="!isUrlField(field) && !isGeoPointField(field)" data-test-field-value [innerHtml]="highlightedInnerHtml(field)"></td>
+                            <td *ngIf="isGeoPointField(field)">
+                                Lat: {{document.fieldValue(field).coordinates[1]}}, Lon: {{document.fieldValue(field).coordinates[0]}}
```

Comment:
It would be more elegant to do this conversion in the typescript part, also to make sure we don't only process fields called `coordinates`.

Reply:
Done, is DocumentViewComponent the place to do this?